pyproject.toml

```diff
@@ -1,7 +1,8 @@
-[project]
+name = "llama-stack-provider-ramalama"
+version = "0.1.0"
 dependencies = [
-    "llama-stack @ git+https://github.com/meta-llama/llama-stack.git@main",
-    "ramalama==0.7.5",
+    "llama-stack>=0.2.2",
```

Comment:
needs to be main, they haven't released with external providers

Reply:
ack, will change this back and open a tracker so we can fix it later

Reply:
https://github.com/containers/llama-stack-provider-ramalama/issues/6